docs/root/configuration/listeners/runtime.rst

```diff
@@ -4,5 +4,5 @@ Runtime
 Listeners support the following runtime settings:
 
 ssl.alt_alpn
-  What % of requests use the configured :ref:`alt_alpn <config_listener_ssl_context_alt_alpn>`
+  *(deprecated)* What % of requests use the configured :ref:`alt_alpn <config_listener_ssl_context_alt_alpn>`
```

Comment:
TBH, I would just delete this entire doc snippet. I doubt anyone is using it and it's basically no longer supported in v2. Thoughts?

Reply:
I was thinking about this yesterday, but my conclusion was that then we have undocumented feature, which doesn't help anybody.

This should be probably removed as part of v1 deprecation process (#2832), along with the code for the feature itself.

Reply:
OK, that's fine with me.